liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/ServerXmlDocument.java

```diff
@@ -42,38 +45,84 @@
     
     private ServerXmlDocument() {    
```

Comment:
Created new ServerConfigDocument to hold configured apps locations. This will cover all your review comments.